public/stylesheets/masters/low_vision_default/README.md

```diff
@@ -0,0 +1,37 @@
+# Low Vision Default Overview
+
+* Takes out all "columns" and makes everything max-width, so dashboard is max,
+filters are max and so it puts the filters at the bottom of the page 
```

Comment:
Maybe "filters are max; puts the filters..." or "filters are max and at the bottom of the page"? Instead of the second "so".

Reply:
This is just reformatted copy pasta from the [skin description](https://archiveofourown.org/skins/891) on production, so I'm just going to fix anything that I messed up in for now and leave the revising for some other time. There are a variety of things that aren't quite right, some of which may also need code changes.